roottest/root/io/datamodelevolution/misc/readTree.C

```diff
@@ -3,9 +3,9 @@ void readTree(const char* filename = "treeTest.root", bool loadSomeClass = true)
 {
 #ifndef ClingWorkAroundMissingDynamicScope
    gROOT->ProcessLine(".L MyClass.cxx+");
-#endif
 	if (loadSomeClass) gROOT->ProcessLine(".L SomeClass.cxx+");
-	
+#endif
```

Comment:
The previous #endif location was intentional.  The script does not use `SomeClass` explicitly in its code and thus the library can be loaded during the script without issues.

Reply:
I move `#endif` on previous place and add extra line to postobj.ref file:
```
Info in <ACLiC>: unmodified script has already been compiled and loaded
```